.github/workflows/pr_test_helm_chart.yml

```
@@ -68,7 +68,7 @@ jobs:
         uses: actions/cache/save@v3
```

As you're using `actions/cache/restore@v4` please use `actions/cache/save@v4` too for consistency of versions.
```suggestion
        uses: actions/cache/save@v4
```